variables.tf

```diff
@@ -115,6 +117,11 @@ locals {
   tf_group_developer_name = "${local.tf_service_name}-${local.stage}-${local.role_developer_name}"
   tf_group_ci_name        = "${local.tf_service_name}-${local.stage}-${local.role_ci_name}"
 
+  # Resolve the name and ARN for either the default or the custom role.
+  default_lambda_role_name = "tf-${var.service_name}-${var.stage}-lambda-execution"
+  lambda_role_name         = "${var.lambda_role_name != "" ? var.lambda_role_name : local.default_lambda_role_name}"
+  lambda_role_arn          = "arn:aws:${local.partition}::${local.account_id}:role/${local.lambda_role_name}"
```

Comment:
Do we want to `REGION`-alize this? Just observing that the default SLS role keeps the region in the name...

Reply:
Thought about this for a bit and how it might affect cross-regional deploys. I think `iam_region = "*"` plus a single instance of the Terraform module/single shared role _should_ be enough?